Automattic/vip-go-mu-plugins
# Refactor Tracks events props to util functions

## Description

- Refactor `Tracks` events props to util functions for easy use.
- Add following properties in each track event

```php
[
	'hosting_provider' => get_hosting_provider(),
	'is_vip_user'      => false,
	'is_multisite'     => is_multisite(),
	'wp_version'       => get_bloginfo( 'version' ),
]
```

## Changelog Description

### Updated

- Refactored Tracks events props to utility functions and added additional global properties.

## Pre-review checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Add following code
    ```php
    function record_telemetry_event_in_footer() {
        $tracks = new \Automattic\VIP\Telemetry\Tracks();
        $tracks->record_event( 'local_event', [
            'hello' => 'vip',
        ] );
    }
    add_action('admin_footer', 'record_telemetry_event_in_footer');
```
2. Load the admin, wait for some time to see the event appear in Tracks and verify the event.
